src/vdaf/prio3.rs

```diff
@@ -349,17 +364,16 @@ where
                 *x -= y;
             }
 
-            helper
-                .joint_rand_param
-                .seed_hint
-                .xor_accumulate(&joint_rand_seed);
+            helper.joint_rand_param.seed_hint = Vec::with_capacity(num_aggregators as usize - 1);
+            let hint = &mut helper.joint_rand_param.seed_hint;
+            hint.push(leader_joint_rand_seed_part.clone());
+            hint.extend(helper_joint_rand_parts[..j].iter().cloned());
+            hint.extend(helper_joint_rand_parts[j + 1..].iter().cloned());
         }
 
-        leader_joint_rand_seed_hint.xor_accumulate(&joint_rand_seed);
-
         let leader_joint_rand_param = if self.typ.joint_rand_len() > 0 {
             Some(JointRandParam {
-                seed_hint: leader_joint_rand_seed_hint,
+                seed_hint: helper_joint_rand_parts.clone(),
```

Comment:
Is this clone really needed? I wonder if we can re-arrange so that the borrow checker is happy with a move here.